plugins/com.google.cloud.tools.eclipse.util/src/com/google/cloud/tools/eclipse/util/ArtifactRetriever.java

```diff
@@ -193,6 +169,16 @@ private ArtifactVersion getLatestIncrementalVersion(String coordinates, VersionR
     }
   }
 
+  private static ArtifactVersion getLatestReleasedVersion(
+      NavigableSet<ArtifactVersion> allVersions) {
+    for (ArtifactVersion version : allVersions.descendingSet()) {
```

Comment:
Just checking in. Does this sorting works correctly as below?

- "19.0" comes before "2.0".
- "1.0.0" comes before "1.0.0-SNAPSHOT".